.github/workflows/deploy-c4-diagrams.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy C4 Diagrams
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
I just checked https://planning-inspectorate.github.io/appeal-planning-decision/ before I realised that this will only run once we merge into `main` 🤞